api/src/main/java/org/apache/cloudstack/api/command/admin/zone/UpdateZoneCmd.java

```diff
@@ -95,6 +95,9 @@
     @Parameter(name = ApiConstants.LOCAL_STORAGE_ENABLED, type = CommandType.BOOLEAN, description = "true if local storage offering enabled, false otherwise")
     private Boolean localStorageEnabled;
 
+    @Parameter(name = ApiConstants.SORT_KEY, type = CommandType.INTEGER, description = "sort key of the disk offering, integer")
```

Comment:
this is not the sortkey for the diskoffering, is it?

Reply:
Yes @DaanHoogland , I'll fix.